various-receipes/nestjs/app/app.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller()
+export class AppController {
+  constructor() {}
+
+  @Get('/hello')
+  getHello() {
+    return {
```

Comment:
Maybe add comment that clarifies that "Here we just exemplifies a simple route and the setup of Nest.js. To learn about testing patterns of real-world app, look at the main example under..."